enterprise/server/api/api_server.go

```diff
@@ -38,6 +40,7 @@ var (
 	enableAPI            = flag.Bool("api.enable_api", true, "Whether or not to enable the BuildBuddy API.")
 	enableCache          = flag.Bool("api.enable_cache", false, "Whether or not to enable the API cache.")
 	enableCacheDeleteAPI = flag.Bool("enable_cache_delete_api", false, "If true, enable access to cache delete API.")
+	enableMetricsAPI     = flag.Bool("enable_metrics_api", true, "If true, enable access to metrics API.")
```

Comment:
should we make this api.enable_metrics_api? I'm not sure why the previous flag does not conform.